### ISSUE TITLE
Update Chapter 9

### ISSUE THREAD

**Issue by dwhitney** [purescript-contrib/purescript-book#26]

**Comment by milesfrain** [comment]
PR for updates in this chapter in #51 

For reference, here's are [solutions](https://github.com/milesfrain/purescript-book/commit/511767f451fe977f06244a71cc56bef91f89da1e) for that chapter. I have this setup with adjacent `.html` and `.js` files in `src` to enable automatic reloads upon IDE save with `parcel` like so:
```
parcel src/plots.html --open                                                                  
parcel src/lsystem.html --open
```
I think this workflow is much nicer than the manual `spago bundle-app` + `browser refresh` when working on exercises.
Should we create the extra files to enable this workflow for the exercises? A downside is that it introduces another dependency (`parcel`).

**Comment by milesfrain** [comment]
This issue should be closed now that #51 is merged.
